.github/workflows/appengine-deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: App Engine Deploy Integration
+
+on: 
+  push:
+    branches:
+      - main 
+  pull_request:
+
+env:
+  APPENGINE_VERSION: invalid
+
+jobs:
+  appengine-deploy:
+    name: App Engine Deploy Integration
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - id: vars
+        name: Declare sha_short as a variable
```

Comment:
Is the _variable_ term correct? Based on [1] it seems to be called _output parameter_.

[1] https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter